workflow/scripts/test/test_content.py

```diff
@@ -0,0 +1,372 @@
+        "Load",
+        "load_tx",
+        bus="bus3",
+        p_set=pd.Series(300, index=n.snapshots),
+    )
+
+    return n
+
+
+@pytest.fixture
+def tct_test_network():
+    """Create a network for testing Technology Capacity Target constraints."""
+    # Create a simple network for testing
+    n = pypsa.Network()
+
+    # Add snapshots and investment periods
+    n.set_snapshots(pd.date_range("2030-01-01 00:00", "2030-01-01 23:00", freq="H"))
+    n.set_investment_periods(["2030"])
+
+    # Add buses
+    n.add("Bus", "bus1", x=0, y=0, country="US", region="west")
+    n.add("Bus", "bus2", x=1, y=1, country="US", region="west")
+    n.add("Bus", "bus3", x=2, y=2, country="US", region="east")
+
+    # Add technology types that would be targeted by TCT constraints
+    n.add(
+        "Generator",
+        "wind_west1",
+        bus="bus1",
+        p_nom=0,
+        p_nom_extendable=True,
+        carrier="onwind",
+        capital_cost=1000,
+        marginal_cost=0.1,
+        p_max_pu=pd.Series(0.8, index=n.snapshots),
+    )
+
+    n.add(
+        "Generator",
+        "wind_west2",
+        bus="bus2",
+        p_nom=0,
+        p_nom_extendable=True,
+        carrier="onwind",
+        capital_cost=1100,
+        marginal_cost=0.12,
+        p_max_pu=pd.Series(0.75, index=n.snapshots),
+    )
+
+    n.add(
+        "Generator",
+        "wind_east",
+        bus="bus3",
+        p_nom=0,
+        p_nom_extendable=True,
+        carrier="onwind",
+        capital_cost=950,
+        marginal_cost=0.09,
+        p_max_pu=pd.Series(0.85, index=n.snapshots),
+    )
+
+    # Add storage that could be targeted by TCT
+    n.add(
+        "StorageUnit",
+        "battery_west",
+        bus="bus1",
+        p_nom=0,
+        p_nom_extendable=True,
+        carrier="battery",
+        capital_cost=300,
+        marginal_cost=0.01,
+        efficiency_store=0.95,
+        efficiency_dispatch=0.95,
+        standing_loss=0.01,
+        max_hours=6,
+    )
+
+    # Add other generation technologies
+    n.add(
+        "Generator",
+        "gas_west",
+        bus="bus2",
+        p_nom=100,
+        p_nom_extendable=True,
+        carrier="gas",
+        capital_cost=500,
+        marginal_cost=30,
+        p_max_pu=pd.Series(1.0, index=n.snapshots),
+    )
+
+    n.add(
+        "Generator",
+        "gas_east",
+        bus="bus3",
+        p_nom=100,
+        p_nom_extendable=True,
+        carrier="gas",
+        capital_cost=450,
+        marginal_cost=28,
+        p_max_pu=pd.Series(1.0, index=n.snapshots),
+    )
+
+    # Add loads
+    n.add(
+        "Load",
+        "load_west1",
+        bus="bus1",
+        p_set=pd.Series(200, index=n.snapshots),
+    )
+
+    n.add(
+        "Load",
+        "load_west2",
+        bus="bus2",
+        p_set=pd.Series(150, index=n.snapshots),
+    )
+
+    n.add(
+        "Load",
+        "load_east",
+        bus="bus3",
+        p_set=pd.Series(250, index=n.snapshots),
+    )
+
+    return n
```

Comment:
Why cant you use the base network here, and build onto it like in the other tests?